### PR TITLE
[DEV-868] Hosting landing page

### PR DESCRIPTION
**Pull request by carlosabadia**
@picklelo added the todos for the sections that needs the wording

Also, @Alek99 what do we do with the banner, remove it? Change the wording to "Join now" or smthing?

### CODE REVIEW

**Comment by tgberkeley** [comment]
@carlosabadia I made an update to the table for all the new information that we want. I have 4 small fixes to make to this pricing page as well.

Screenshot 2024-12-02 at 7 22 22 PM

In the picture above we need to make the dedicated channel text fit nicely in the page, I guess by making the table row have a larger height. 

We need to make the buttons fit more appropriately as we change the size of the screen.

The Contact Sales buttons should not open an email, they should link to where the users go to fill out the enterprise demo form that we had on the website previously. We should bring this enterprise demo form back on the pricing page.

We should remove the price calculator for team as this is no longer needed or relevant as we make them contact us for this now.

**Comment by tgberkeley** [comment]
We should update the Hobby, Pro, Team and Enterprise plan cards with this information instead and we need to make it really clear that hobby is free
Screenshot 2024-12-02 at 7 33 36 PM

**Comment by carlosabadia** [comment]
@tgberkeley 
Make that long text fit makes the row look weird... Can we just instead just say Dedicated support and add a row on that section like this:
image

Also, buttons looks fine to me. Let me check on safari

**Comment by carlosabadia** [comment]
@tgberkeley added the changes, feel free to change any text/icons. We already say on the hobby button to start for free, I guess we can mention it again that is free on the plan description if you want to

**Comment by tgberkeley** [comment]
Nice I added all my final small changes. Everything looks great now. Only small thing is that the buttons still look a little weird on my browser, but other than that it all looks great.

File: pcweb/pages/hosting/views/deploy_animation.py [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+from pcweb.pages.hosting_countdown.animated_box import (
+    terminal_box,
+    deploy_box,
+    typing_text_script,
+)
+
+
+def grid() -> rx.Component:
+    return rx.html(
+        """<svg width="1028" height="320" viewBox="0 0 1028 320" fill="none" xmlns="http://www.w3.org/2000/svg">
```
[the remark this answers is not in the document]
Comment (carlosabadia) [reply pcweb/pages/hosting/views/deploy_animation.py:11]: only happens on dev due to the on_mount event triggering twice

**Review by picklelo — APPROVED** [review]

**Pull request merged**